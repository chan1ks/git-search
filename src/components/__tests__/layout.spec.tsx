import React from 'react';

import renderer from 'react-test-renderer';
import Layout from '../__layout/layout';

describe('Layout', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Layout children={[]} title="This is a title!" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
