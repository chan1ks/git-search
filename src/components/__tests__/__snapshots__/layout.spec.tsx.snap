// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  className="flex column full-height"
>
  <div
    className="typography container-full layout--center full-width grow-1"
  >
    <h1
      style={
        Object {
          "marginBottom": "10px",
        }
      }
    >
      This is a title!
    </h1>
    <main />
  </div>
  <div
    className="layout--footer"
  >
    <footer>
      2020
       | Made with ‚ù§ by
       
      <a
        href="https://github.com/chan1ks"
      >
        chan1ks
      </a>
       | Built with
       
      <a
        href="https://www.gatsbyjs.com"
      >
        Gatsby
      </a>
    </footer>
  </div>
</div>
`;
